pkg/mutation/assign_mutator.go

```diff
@@ -39,6 +41,14 @@ func (m *AssignMutator) SchemaBindings() []SchemaBinding {
 	return m.bindings
 }
 
+func (m *AssignMutator) Value() interface{} {
+	assign := make(map[string]interface{})
+	if err := json.Unmarshal(m.assign.Spec.Parameters.Assign.Raw, &assign); err == nil {
+		return assign
+	}
+	return string(m.assign.Spec.Parameters.Assign.Raw)
```

Comment:
Not sure about falling back to this, maybe it's better to return an error?

Reply:
Error added.